src/engine-actions/close-channels.js

```diff
@@ -0,0 +1,70 @@
+const {
+  listChannels,
+  closeChannel
+} = require('../lnd-actions')
+/**
+ * Closes a channel given the channel point
+ *
+ * @param {Boolean} [options.outbound=true] outbound is true if checking outbound channels, false if inbound
+ * @returns {Promise<Array.<Object>>}
+ */
+async function closeChannels ({ force = false } = {}) {
+  const { channels = [] } = await listChannels({ client: this.client })
+  const activeChannels = channels.filter(c => c.active)
+
+  const closedChannelResponses = await Promise.all(activeChannels).map(channel => close(channel, force, this.client, this.logger))
```

Comment:
[nit] Instead of passing in client/logger to this function, we could use `close.call(this, channel, force)` which would do the same thing by binding the current context to the call.